exec.go

```diff
@@ -86,7 +87,7 @@ following will output a list of processes running in the container:
 		},
 	},
 	Action: func(context *cli.Context) error {
-		if err := checkArgs(context, 2, minArgs); err != nil {
+		if err := checkArgs(context, 1, minArgs); err != nil {
```

Comment:
This should actually be conditional based on whether `-p` was provided. I'll write a patch to fix it.